docker/sysdig/Dockerfile

```diff
@@ -1,26 +1,12 @@
-FROM ghcr.io/draios/sysdig-builder:dev AS builder
+FROM ghcr.io/draios/sysdig-builder:dev
```

Comment:
Why are we using the builder as base image, which is based on the older CentOS 7? Is there a specific reason to avoid UBI8 (or anyways a more modern image)?

The reason why the builder appears to be a bit older is that we want to be as compatible as we can with the glibc version used to link the final binaries.

Reply:
Yeah, you are right! It's better to keep ubi8!